toonz/sources/toonz/xsheetviewer.cpp

```diff
@@ -397,6 +411,18 @@ void XsheetViewer::positionSections() {
                         bodyLayer.adjusted(0, -XsheetGUI::SCROLLBAR_WIDTH)));
   m_rowScrollArea->setGeometry(o->frameLayerRect(
       bodyFrame.adjusted(0, -XsheetGUI::SCROLLBAR_WIDTH), headerLayer));
+  if (o->isVerticalTimeline())
+    m_frameZoomSlider->hide();
+  else {
+    if (m_frameZoomSlider->isHidden()) m_frameZoomSlider->show();
+    QRect sliderRect = QRect(0, 0, XsheetGUI::ZOOM_SLIDER_WIDTH,
+                             XsheetGUI::SCROLLBAR_WIDTH - 2)
+                           .translated(m_columnScrollArea->rect().right() -
+                                           XsheetGUI::ZOOM_SLIDER_WIDTH - 17,
+                                       m_noteScrollArea->height() +
+                                           m_columnScrollArea->height());
```

Comment:
For now the slider seems to be placed irregularly if you toggle the Xsheet toolbar to visible.
How about modifying the vertical position of `sliderRect` from 
`m_noteScrollArea->height() + m_columnScrollArea->height()` to
`height() - XsheetGUI::SCROLLBAR_WIDTH + 1`  ?

Reply:
@shun-iwasawa Thanks for catching that. I've modified the logic per suggestion and verified it worked properly.